frontend/pages/policies/ManagePoliciesPage/components/RunScriptModal/RunScriptModal.tsx

```diff
@@ -0,0 +1,241 @@
+    [
+      {
+        scope: "scripts",
+        team_id: teamId,
+      },
+    ],
+    ({ queryKey: [queryKey] }) =>
+      scriptsAPI.getScripts(omit(queryKey, "scope")),
+    {
+      select: (data) => data.scripts,
+      ...DEFAULT_USE_QUERY_OPTIONS,
+    }
+  );
+
+  const onUpdate = useCallback(() => {
+    onSubmit(formData);
+  }, [formData, onSubmit]);
+
+  const onChangeEnableRunScript = useCallback(
+    (newVal: { policyName: string; value: boolean }) => {
+      const { policyName, value } = newVal;
+      setFormData(
+        formData.map((policy) => {
+          if (policy.name === policyName) {
+            return {
+              ...policy,
+              runScriptEnabled: value,
+              scriptIdToRun: value ? policy.scriptIdToRun : undefined,
+            };
+          }
+          return policy;
+        })
+      );
+    },
+    [formData]
+  );
+
+  const onSelectPolicyScript = useCallback(
+    ({ name, value }: IScriptDropdownField) => {
+      const [policyName, scriptId] = [name, value];
+      setFormData(
+        formData.map((policy) => {
+          if (policy.name === policyName) {
+            return { ...policy, scriptIdToRun: scriptId };
+          }
+          return policy;
+        })
+      );
+    },
+    [formData]
+  );
+
+  const availableScriptOptions = availableScripts?.map((script) => ({
+    label: script.name,
+    value: script.id,
+  }));
+
+  const renderPolicyRunScriptOption = (policy: IFormPolicy) => {
+    const {
+      name: policyName,
+      id: policyId,
+      runScriptEnabled: enabled,
+      scriptIdToRun,
+    } = policy;
+
+    return (
+      <li
+        className={`${baseClass}__policy-row policy-row`}
+        id={`policy-row--${policyId}`}
+        key={`${policyId}-${enabled}`}
+      >
+        <Checkbox
+          value={enabled}
+          name={policyName}
+          onChange={() => {
+            onChangeEnableRunScript({
+              policyName,
+              value: !enabled,
+            });
+          }}
+        >
+          <TooltipTruncatedText value={policyName} />
+        </Checkbox>
+        {enabled && (
+          <Dropdown
+            options={availableScriptOptions}
+            value={scriptIdToRun}
+            onChange={onSelectPolicyScript}
+            placeholder="Select script"
+            className={`${baseClass}__script-dropdown`}
+            name={policyName}
+            parseTarget
+          />
+        )}
+      </li>
+    );
+  };
+
+  const renderContent = () => {
+    if (isAvailableScriptsError) {
+      return <DataError />;
+    }
+    if (isLoadingAvailableScripts) {
+      return <Spinner />;
+    }
+    if (!availableScripts?.length) {
+      return (
+        <div className={`${baseClass}__no-scripts`}>
+          <b>No scripts available for install</b>
+          <span>
+            Go to <b>Controls &gt; Scripts</b> to add scripts to this team.
```

Comment:
Could we link this?

Reply:
Great idea. This was not specced, but mimics the similar empty state for the Install software automation modal, which was specced without links. @noahtalerman wdyt of making this and the comparable install software empty state actually clickable links?

Reply:
👌 https://github.com/fleetdm/fleet/pull/22436/commits/c2a59aadc387acafcabd4cd55037466ce3b78fce

![ezgif-2-12035f476c](https://github.com/user-attachments/assets/c5f761f3-e16d-4bfe-a245-2a684e17e701)

(and same for policy automations empty state)

Reply:
Nice! Lots less fiddly than bouncing to that part of the UI manually and potentially losing the team you were working with in the process.

Reply:
Looks great! Maybe we add the same (except link to software page) in the empty state for the Install software automation modal?

Reply:
Already did it @noahtalerman